### PR TITLE
FIX: Not all fields in the place property are sent using Ninjs formatter. [SDESK-5080]

### PR DESCRIPTION
**Pull request by GyanP**
I will set `NINJS_PLACE_EXTENDED` to False before merging.

### CODE REVIEW

**Comment by petrjasek** [comment]
I'm not sure we should just change the output format, that might be problematic for receiving systems. so we would either need a config for it, or even use a custom format for publisher as this is not respecting the specs

**Comment by GyanP** [comment]
I think having a config like `SEND_ALL_FIELDS_FOR_PLACE_IN_OUTPUT` and based on this decide to what to send. Or if we plan to create a custom format for a publisher then how to decide whether this is for the publisher or other?

**Comment by petrjasek** [comment]
what about `NINJS_PLACE_EXTENDED` setting? it's just ninjs not all outputs

File: superdesk/default_settings.py [hunk 1]
```diff
@@ -834,3 +834,6 @@ def local_to_utc_hour(hour):
 TANSA_LICENSE_KEY = env('TANSA_LICENSE_KEY')
 TANSA_CLIENT_BASE_URL = env('TANSA_CLIENT_BASE_URL', 'https://d02.tansa.com/tansaclient/')
 TANSA_PROFILES = {}
+
+# Enable ninjs to send all the fields for place in output.
+NINJS_PLACE_EXTENDED = True
```
Comment (petrjasek) [inline superdesk/default_settings.py]: would make it `False` by default so we don't change behaviour

Comment (GyanP) [reply superdesk/default_settings.py]: I will set it to false before merging. I already mentioned in the first comment.

**Pull request merged**